src/glossary.rst

```diff
@@ -691,6 +691,11 @@ Glossary
       See also:
 
       * :term:`cold start`
+   
+   source directory
+      An arbitrary directory where workflows are written and stored in
+      preparation for installation with `cylc install` or reinstallation with
```

Comment:
```suggestion
      preparation for installation with ``cylc install`` or reinstallation with
```

Need to add some sort of a check for these, such an easy mistake to make. I bet there are loads we've missed.